charts/sysdig/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sysdig
-version: 1.12.23
+version: 1.12.24
```

Comment:
@endriu0  please rebase and submit the PR. The current version is already 1.12.24 and appVersion has changed to 12.0.2.

Reply:
Done - thanks @balinaveen